Controller/Adminhtml/Configuration/WebhookTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Payment Module
+ *
+ * Copyright (c) 2022 Adyen N.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+
+namespace Adyen\Payment\Controller\Adminhtml\Configuration;
+
+use Adyen\Payment\Helper\ManagementHelper;
+use Magento\Backend\App\Action;
+use Magento\Backend\App\Action\Context;
+use Magento\Framework\Controller\Result\JsonFactory;
+
+class WebhookTest extends Action
+{
+    /**
+     * @var ManagementHelper
+     */
+    private $managementApiHelper;
+
+    /**
+     * @var \Adyen\Payment\Helper\Data
+     */
+    protected $adyenHelper;
+
+    /**
+     * @var JsonFactory
+     */
+    protected $resultJsonFactory;
+
+    public function __construct(
+        Context $context,
+        ManagementHelper $managementApiHelper,
+        \Adyen\Payment\Helper\Data $adyenHelper,
+        JsonFactory $resultJsonFactory
+    ) {
+        parent::__construct($context);
+        $this->managementApiHelper = $managementApiHelper;
+        $this->adyenHelper =$adyenHelper;
+        $this->resultJsonFactory = $resultJsonFactory;
+    }
+
+    /**
+     * @return \Magento\Framework\App\ResponseInterface|\Magento\Framework\Controller\ResultInterface|mixed|string
+     * @throws \Magento\Framework\Exception\NoSuchEntityException
```

Comment:
Can we import the namespaces and add a return type to the function?

Reply:
interface definition does not declare a return type so we have to leave it out here too